Improve dashboard UI

CoryMortimer opened the pull request:
On the dashboard page, added loading animation and animation to all
cards when they appear. Changed the top 3 cards to have an image. Added
more information to the recent news section. Also added the view more
button.

Added animation to news page.

Added animation to news article page and changed the layout to card
over.

In the news creator, added animation to when the page first loads.
Changed the stepper to be vertical and to follow the teradata
recommendations. Added a loading animation when submitting and then a
submit message.

Resolves #16

CoryMortimer committed:
Improve dashboard UI

On the dashboard page, added loading animation and animation to all
cards when they appear. Changed the top 3 cards to have an image. Added
more information to the recent news section. Also added the view more
button.

Added animation to news page.

Added animation to news article page and changed the layout to card
over.

In the news creator, added animation to when the page first loads.
Changed the stepper to be vertical and to follow the teradata
recommendations. Added a loading animation when submitting and then a
submit message.

Resolves #16

JulieMarie commented on src/app/dashboard/dashboard.component.html:
```diff
@@ -34,15 +38,19 @@
           <mat-divider></mat-divider>
           <app-articles
             [maxArticles]="maxArticles">
           </app-articles>
```
There's a horizontal line missing here that should separate the list from the card actions (https://teradata.github.io/covalent/#/design-patterns)

JulieMarie commented on src/testing/index.ts:
```diff
@@ -42,6 +42,7 @@ export const stocks = {
   stocks: [
     {
       name: 'Dow Jones Industrial Average',
+      image: 'https://upload.wikimedia.org/wikipedia/en/thumb/f/f8/Dow_Jones_Logo.svg/1200px-Dow_Jones_Logo.svg.png',
```
The images need some padding around them.  Check out the padding utility styles here: https://teradata.github.io/covalent/#/style-guide/utility-styles

JulieMarie commented on src/app/articles/articles.component.html:
```diff
@@ -1,8 +1,9 @@
 <mat-list>
```
Maybe turn the list into another `mat-card` with an action button (CREATE) at the bottom to that goes to the news creator?

CoryMortimer replied:
Are you referring to the news page? This component is put in a card on the dashboard page and just displayed on the news page. I could make it into a card on the news page, maybe using the card over layout?

JulieMarie replied:
Yep to the News page here:
![screen shot 2018-10-18 at 9 52 35 pm](https://user-images.githubusercontent.com/437977/47198276-2e0fc600-d320-11e8-8884-033ddda1b7f3.png)

I'm suggesting putting the list of news items into a `mat-card`....maybe something like this:
![screen shot 2018-10-18 at 9 54 27 pm](https://user-images.githubusercontent.com/437977/47198335-79c26f80-d320-11e8-8697-c57c9ccab419.png)
with a `CREATE` action at the bottom like:
![screen shot 2018-10-18 at 9 55 28 pm](https://user-images.githubusercontent.com/437977/47198375-aaa2a480-d320-11e8-9e08-39510ff85f95.png)
Does that make sense?  It just looks awkward taking up the full width like that.

JulieMarie reviewed (changes requested):
Looking good ... left some suggestions!!

CoryMortimer committed:
Add horizontal line and padding around images

Add a horizontal line to the Recent News card dividing the news articles
from the button. Also, add some padding around the images in the stock
cards.

CoryMortimer committed:
Turn news page into a card layout

The news page was just a list on a page. Add the card layout to make it
more appealing. Also, add a link to the news creator page from the news
page.

CoryMortimer commented:
@JulieMarie Hopefully, I have addressed your comments. Thanks for the feedback! I wasn't quite sure what you meant by the comment on articles.component.html so please let me know if I misinterpreted that request. That feature is part of commit 22ca10d.

JulieMarie reviewed (approved):
Requested changes made...awesome!!
